app/views/_container-statuses.html

```diff
@@ -0,0 +1,74 @@
+<!--
+  Expects the following variables:
+  containerStatuses
+  startDebugTerminal (optional)
+  detailed (optional)
+ -->
+
+<div ng-if="detailed && pod.status.initContainerStatuses.length">
+  <h4 row>
+    <span><i class="fa" ng-class="{'fa-check text-success': initContainersTerminated}"></i></span>
+    <span flex>
+      <span ng-if="initContainersTerminated"> &nbsp;{{pod.status.initContainerStatuses.length}}</span>
+      <ng-pluralize count="pod.status.initContainerStatuses.length"
+                    when="{'1': 'Init Container','other': 'Init Containers'}">
+      </ng-pluralize>
+      <span ng-if="initContainersTerminated">completed successfully</span>
+    </span>
+    <span ng-if="initContainersTerminated">
+      <a class="page-header-link" href="" ng-click="toggleInitContainer()" ng-if="!expandInitContainers">Show Details</a>
+      <a class="page-header-link" href="" ng-click="toggleInitContainer()" ng-if="expandInitContainers">Hide Details</a>
```

Comment:
Suggest one `a` element and just change the message inside since everything else is the same.

```html
<a class="page-header-link" href="" ng-click="toggleInitContainer()">
  <span ng-if="!expandInitContainers">Show</span>
  <span ng-if="expandInitContainers">Hide</span>
  Details
</a>
```